pkgs/build-support/setup-hooks/auto-patchelf.py

```diff
@@ -247,20 +248,25 @@ def auto_patchelf_file(path: Path, runtime_deps: list[Path], append_rpaths: List
     # failing at the first one, because it's more useful when working
     # on a new package where you don't yet know the dependencies.
     for dep in file_dependencies:
-        if dep.is_absolute() and dep.is_file():
-            # This is an absolute path. If it exists, just use it.
-            # Otherwise, we probably want this to produce an error when
-            # checked (because just updating the rpath won't satisfy
-            # it).
-            continue
+        found_dependency = None
+        for candidate in dep:
+            if candidate.is_absolute() and candidate.is_file():
+                # This is an absolute path. If it exists, just use it.
+                # Otherwise, we probably want this to produce an error when
+                # checked (because just updating the rpath won't satisfy
+                # it).
```

Comment:
I think this file is formatted at 100 columns wide.
```suggestion
                # checked (because just updating the rpath won't satisfy it).
```